extensions/2.0/Vendor/FB_geometry_metadata/schema/worldBounds.schema.json

```diff
@@ -0,0 +1,36 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "minimum and maximum bounding box extent",
+    "type": "object",
+    "allOf": [
+        {
+            "$ref": "glTFProperty.schema.json"
+        }
+    ],
+    "properties": {
+        "min": {
+            "type": "array",
+            "items": {
+                "type": "number"
+            },
+            "description": "The bounding box corner with the numerically lowest world-space coordinates",
+            "minItems": 3,
+            "maxItems": 3
+        },
+        "max": {
+            "type": "array",
+            "items": {
+                "type": "number"
+            },
+            "description": "The bounding box corner with the numerically highest world-space coordinates",
```

Comment:
Here and above, would it be any clearer to say *scene-space* rather than *world-space*?  These numbers are expressed in terms of the glTF scene, not any particular node, correct?  I'm trying to accommodate the idea that the glTF itself may not be at the center of world space in whatever client has loaded it.

Reply:
That's a good point. I don't know if _scene space_ is an understood notion, but I suppose it's fairly self-explanatory!